### ISSUE TITLE
Improve documentation of light color schemes

### ISSUE THREAD

**Issue by GloverDonovan** [dylanaraps/pywal#323]
Hi,

When a color scheme has a light and dark variant, it isn't immediately obvious that using the name of the light variant will select the dark color scheme unless `-l` is specified. I think this should be explained in more places, such as when the user uses `--theme` to view the available themes.

It may also be worth mentioning that not all dark themes have light themes, and not all light themes have dark themes.

wal 3.1.0

**Comment by Migacz85** [comment]
Yup I can agree :-) How exactly the -l flag is working some example to produce 'light' theme? I would like to have some light background and darker fonts on it during the day, and opposite in the night. Any idea how to run the command to have this effect on?
